sdks/python/apache_beam/runners/google_cloud_dataflow/internal/clients/dataflow/__init__.py

```diff
@@ -20,8 +20,8 @@
 
 import pkgutil
 
+from apitools.base.py import *
```

Comment:
I will remove it, I do not think there is a reason.

This line was removed in the previous PR and I noticed that other clients have it to. It might be auto-generated, not sure.